scan-car/src/components/navigation.css

```diff
@@ -0,0 +1,59 @@
+header {
+    background-color: #20334B;
```

Comment:
Define the colors globally using CSS variables. (See [https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties]) It prevents typos if you have to write color names rather than the HEX-values.

Reply:
@mathildegeh 
@marinabjorns 
Let's discuss this again on Monday